jgrapht-core/src/test/java/org/jgrapht/alg/lca/NaiveLCAFinderTest.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * (C) Copyright 2016-2018, by Barak Naveh and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.alg.lca;
+
+import org.jgrapht.Graph;
+import org.jgrapht.graph.DefaultDirectedGraph;
+import org.jgrapht.graph.DefaultEdge;
+import org.jgrapht.graph.SimpleDirectedGraph;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Set;
+
+/**
+ * Tests for the {@link NaiveLCAFinder}
+ *
+ * @author Barak Naveh
+ * @author Alexandru Valeanu
+ */
+public class NaiveLCAFinderTest {
```

Comment:
Can you also extends your base test class in order to also run your new tests?

Reply:
You should also remove the old test class `org.jgrapht.alg.NaiveLcaFinderTest.java`

Reply:
Not really. `LCATestBase` contains tests that are not suitable for a naive implementation as they will not terminate.

Reply:
Then it's probably worth splitting to distinguish the small cases from the large ones; either that or use an explicit suite to select them.